test/e2e/kubelet_stats.go

```diff
@@ -661,3 +695,36 @@ func (r *resourceMonitor) GetCPUSummary() nodesCPUSummary {
 	}
 	return result
 }
+
+func (r *resourceMonitor) GetMasterNodeCPUSummary(summaryPerNode nodesCPUSummary) nodesCPUSummary {
+	result := make(nodesCPUSummary)
+	var masterSummary containersCPUSummary
+	var nodesSummaries []containersCPUSummary
+	for node, summary := range summaryPerNode {
```

Comment:
For this problem, in fact there are some other places need to be fixed, just mark here:
```
lantaol@lantaol0:~/workspace/src/k8s.io/kubernetes/test/e2e$ grep -ir "Suffix(node" *
kubelet_stats.go:		if strings.HasSuffix(node, "master") {
kubelet_stats.go:		if strings.HasSuffix(node, "master") {
metrics_grabber_test.go:			if strings.HasSuffix(node.Name, "master") {
metrics_grabber_test.go:			if strings.HasSuffix(node.Name, "master") {
metrics_util.go:		if strings.HasSuffix(node.Name, "master") {
```